src/batches/batches.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { CreateBatchDto } from './dto/create-batch.dto';
+//import { UpdateBatchDto } from './dto/update-batch.dto';
+import { PrismaService } from './../prisma/prisma.service';
+
+@Injectable()
+export class BatchesService {
+  constructor(private prisma: PrismaService) {}
+
+  create(createBatchDto: CreateBatchDto) {
+    return this.prisma.batch.create({ data: createBatchDto });
```

Comment:
We shouldn't directly pass the DTO to database writer, we should create a database entity object and pass that instead. For now, these both wil look more or less similar, DTO will have these annotations.

Reply:
Alright, I will map the properties from createBatchDto to the corresponding properties in a database entity object and pass that.